README.md

```diff
@@ -36,6 +36,11 @@ on each node to the `~/.ssh/authorized_keys` file for the user set by the `clust
 Launching an EC2 cluster
 ------------------------
 
+When Zetten launches a cluster, it uses free CentOS 7 image that is hosted in the AWS marketplace but managed
+by the CentOS orginization.  If you have never used this image in EC2 before, you will need to go to the 
+[Centos 7 product page][centos7] to accept the software terms under the 'Manual Launch' tab.  If this is not 
+done, you will get an error when you try to launch your cluster.
```

Comment:
Could also mention that periodically the centos org will update ami's and deperecate older ami's, making those older ami's unavailable to new users.